docs/error-reporting/advanced/custom-symbol-server.md

```diff
@@ -14,9 +14,11 @@ Backtrace integrations using minidumps require symbol files to generate valuable
 To meet Backtrace's requirements, the custom symbol server must:
 
 - Follow Microsoft's standard symbol structure (that is, `<url>/<object_name>/<debug_id>/<file>`).
-- Use tools like Windows symstore to generate files in the required structure.
+-- Use tools like Windows symstore to generate files in the required structure.
```

Comment:
Double dashes is not an indented bullet, which I think is what you are trying to achieve? This is the result of double dashes.
![image](https://github.com/saucelabs/sauce-docs/assets/115846221/19db217c-c9ca-456b-9c02-1628b1449996)

Use spacespacedash for an indented bullet.
![image](https://github.com/saucelabs/sauce-docs/assets/115846221/aa3ec5a7-d9e9-421a-8fa9-f46940d3c493)

```suggestion
  - Use tools like Windows symstore to generate files in the required structure.
```